default-ecr-lifecycle-policy.json

```diff
@@ -7,7 +7,7 @@
         "tagStatus": "tagged",
         "tagPrefixList": ["master", "main"],
         "countType": "imageCountMoreThan",
-        "countNumber": 1000
+        "countNumber": 100
```

Comment:
I feel like 1k on the main branch is reasonable. I'm not sure, but I'd guess a lot of our cost is coming from the other 5k on feature branches.